Do not check for SDL_ttf version because it fails when
cross compiling.

--- a/configure.ac	2014-12-19 22:26:06.000000000 +0100
+++ b/configure.ac	2015-11-19 12:08:46.996789539 +0100
@@ -176,19 +176,6 @@
 dnl ----------------------------------------
 dnl Check for SDL_ttf library
 dnl ---------------------------------------
-if test "$MINGW32" = no; then
-  AC_MSG_CHECKING([for SDL_ttf >=2.0.6])
-  AC_RUN_IFELSE([AC_LANG_SOURCE(
-      [[#include <SDL_ttf.h>
-        int main(int argc, char *argv[]) {
-        if (TTF_MAJOR_VERSION < 2)
-          return 1;
-        else if (TTF_MAJOR_VERSION == 2 && TTF_MINOR_VERSION == 0 && TTF_PATCHLEVEL < 6) 
-          return 1;
-        return 0;}]])],
-    [AC_MSG_RESULT([found])],
-    [AC_MSG_ERROR([SDL_ttf >= 2.0.6 not found.])])
-fi
 AC_CHECK_LIB(SDL_ttf, main,,[AC_MSG_ERROR([SDL_ttf is required to compile Enigma])])
 
 SDL_LIBS="$SDL_LIBS -lSDL_ttf"
